objcgen/TypeMapper.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Embeddinator
+{
+	public class TypeMapper
+	{
+		Dictionary<ProcessedType, Dictionary<string, ProcessedMemberBase>> MappedTypes = new Dictionary<ProcessedType, Dictionary<string, ProcessedMemberBase>> ();
+		
+		IEnumerable<string> GetNames (ProcessedMemberBase member)
+		{
+			if (member is ProcessedMemberWithParameters) {
+				yield return ((ProcessedMemberWithParameters)member).ObjRawCSignature;
```

Comment:
I did not know about that C# 7 trick, or that we could use such hotness. Fixing.